src/posix/README.md

```diff
@@ -16,70 +16,98 @@ The figure below shows the architecture of OpenThread running in transceiver mod
          POSIX                                          Chip
 ```
 
-## Build
+## Build POSIX CLI
 
 ```sh
-# build core for POSIX
-make -f src/posix/Makefile-posix
-# build transceiver, where xxxx is the platform name, such as cc2538, nrf52840 and so on
-make -f examples/Makefile-xxxx
+./script/cmake-build posix
 ```
 
-## Test
+If built successfully, the binary should be found at: `build/posix/src/posix/ot-cli`.
 
-**NOTE** Assuming the build system is 64bit Linux, you can use the normal OpenThread CLI as described in the [command line document](../../src/cli/README.md). You can also perform radio diagnostics using the command [diag](../../src/core/diags/README.md).
+## Transceivers on different platforms
+
+### Simulation
+
+OpenThread provides an implemenation on simulation platform which enables running a simulated transceiver on the host.
+
+#### Build
+
+```sh
+# Only build RCP so that it goes faster
+./script/cmake-build simulation -DOT_APP_CLI=OFF -DOT_APP_NCP=OFF -DOT_FTD=OFF -DOT_MTD=OFF
+```
+
+#### Run
 
-### With Simulation
+**NOTE** Assuming the build system is 64bit Linux, you can use the normal OpenThread CLI as described in the [command line document](../../src/cli/README.md). You can also perform radio diagnostics using the command [diag](../../src/core/diags/README.md).
 
 ```sh
-make -f examples/Makefile-simulation
-./output/posix/bin/ot-cli 'spinel+hdlc+forkpty://output/simulation/bin/ot-rcp?forkpty-arg=1'
+./build/posix/src/posix/ot-cli 'spinel+hdlc+forkpty://build/simulation/examples/apps/ncp/ot-rcp?forkpty-arg=1'
 ```
 
-### With Real Device
+### nRF52840
 
-#### nRF52840
+The nRF example platform driver has been moved out of OpenThread repo and is placed in a separate repo: [ot-nrf528xx](https://github.com/openthread/ot-nrf528xx). To build transceivers on nRF platform, we need to clone that repo.
```

Comment:
I don't think we need to say the example has moved, let's just state where it is now.

"The nRF example platform driver can be found in the [ot-nrf528xx](https://github.com/openthread/ot-nrf528xx) repo. To build transceivers on the nRF platform, we must clone that repo."